python/cairo-addons/src/cairo_addons/testing/runner.py

```diff
@@ -546,11 +568,19 @@ def _run(entrypoint, *args, verify_squashed_dicts: bool = False, **kwargs):
             if gen_arg_builder is not None
             else lambda _python_type, _value: runner.segments.gen_arg(_value)
         )
-        for i, (arg_name, python_type) in enumerate(
-            [(k, v["python_type"]) for k, v in {**_implicit_args, **_args}.items()]
-        ):
+
+        i = 0
```

Comment:
same question here

Reply:
properly serialize `felt*` (like blake2s_ptr) with no associated args in runner